modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryEpilogueView.kt

```diff
@@ -27,59 +28,81 @@ import androidx.compose.runtime.getValue
+import au.com.shiftyjelly.pocketcasts.endofyear.components.StoryBlurredBackgroundStyle
+import au.com.shiftyjelly.pocketcasts.endofyear.components.StoryFontFamily
 import au.com.shiftyjelly.pocketcasts.endofyear.components.StoryPrimaryText
 import au.com.shiftyjelly.pocketcasts.endofyear.components.StorySecondaryText
 import au.com.shiftyjelly.pocketcasts.endofyear.components.disableScale
 import au.com.shiftyjelly.pocketcasts.repositories.endofyear.stories.StoryEpilogue
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.utils.extensions.dpToPx
+import au.com.shiftyjelly.pocketcasts.utils.featureflag.UserTier
+import com.airbnb.android.showkase.annotation.ShowkaseComposable
 import au.com.shiftyjelly.pocketcasts.localization.R as LR
 
 private val HeartImageSize = 72.dp
-private const val BackgroundColor = 0xFF1A1A1A
 
 @Composable
 fun StoryEpilogueView(
     story: StoryEpilogue,
+    userTier: UserTier,
     onReplayClicked: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
-
-    Column(
-        horizontalAlignment = Alignment.CenterHorizontally,
-        modifier = modifier
+    BoxWithConstraints(
+        modifier = Modifier
             .fillMaxSize()
-            .background(Color(BackgroundColor))
-            .verticalScroll(rememberScrollState())
+            .background(story.backgroundColor)
     ) {
-        Spacer(modifier = modifier.height(40.dp))
-
-        Spacer(modifier = modifier.weight(1f))
-
-        PulsatingHeart()
+        val context = LocalView.current.context
+        StoryBlurredBackground(
+            offset = Offset(
+                -maxWidth.value.toInt().dpToPx(context) * 0.4f,
+                -maxHeight.value.toInt().dpToPx(context) * 0.4f
+            ),
+            style = blurredBackgroundStyle(userTier),
+        )
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = modifier
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState())
+                .padding(vertical = 40.dp)
+        ) {
+            Spacer(modifier = modifier.weight(1f))
 
-        Spacer(modifier = modifier.weight(0.34f))
+            PulsatingHeart()
 
-        PrimaryText(story)
+            Spacer(modifier = modifier.weight(0.34f))
 
-        Spacer(modifier = modifier.weight(0.16f))
+            PrimaryText(story)
 
-        SecondaryText(story)
+            Spacer(modifier = modifier.weight(0.16f))
 
-        Spacer(modifier = modifier.weight(0.32f))
+            SecondaryText(story)
 
-        ReplayButton(onClick = onReplayClicked)
+            Spacer(modifier = modifier.weight(0.16f))
 
-        Spacer(modifier = modifier.weight(1f))
+            ReplayButton(onClick = onReplayClicked)
 
-        PodcastLogoWhite()
+            Spacer(modifier = modifier.weight(1f))
 
-        Spacer(modifier = modifier.height(30.dp))
+            StoryAppLogo()
+        }
+        Confetti {}
```

Comment:
I added this confetti to match it with iOS.

Reply:
I like it! 👍 😄